### DISCUSSION TITLE
Will Pocketbase authentication work for Flutter iOS, android, and web?

### DISCUSSION THREAD

**Discussion by coryfail** [pocketbase/pocketbase#4583] [Q&A]
I come from a Firebase background and with FlutterFire it is relatively easy to add authentication to a Flutter application that is built for both mobile and web. Before I start migrating some of my smaller projects over, I wanted to see if this is something Pocketbase can do. I see that there is a Flutter SDK with Authentication, but from the documentation it doesn't mention if it works for both mobile and web applications. I see an authentication example but it seems too straight forward to work for mobile since other similar BaaS seem to be more complex.

**Comment by ganigeorgiev** [comment]
The supported platforms can be explored on the package page [pub.dev](https://pub.dev/packages/pocketbase).

Also note that the available SDK is not just for Flutter but could be used in non-Flutter project as a regular Dart package, for example in cli applications.

It is built on top of the standard `dart-lang/http` package and it should work on all Dart platforms, with some caveats related to the realtime subscribtions (_and the "all-in-one" OAuth2 flow_) as explained in https://github.com/pocketbase/dart-sdk#limitations.

**Reply by coryfail** [reply]
That's actually really neat, because I have been meaning to mess around with other Dart frameworks. 

How is it that the authentication configuration for Pocketbase so much more straight forward than other similar BaaS?
